### PR TITLE
Converting HTML content to PDF with pdfkit for Python

### PR DESCRIPTION
**Pull request by Anitaachu**
### Article checklist
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by Anitaachu** [comment]
This fixes #1760

**Comment by jumamiller** [comment]
@Anitaachu, I'll review this article
cc @hectorkambow

File: articles/converting-html-content-to-pdf-with-pdfkit-for-python/index.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+**Pdfminer**, **PyMuPDF**, etc are also libraries for converting HTML to PDF in python. However, in this article, we will be using **pdfkit**. Pdfkit is one of the better approaches as, it renders HTML into PDF with various image formats, HTML forms, and other complex printable documents.
+
+**N.B:** There are also libraries and tools for converting PDF in Django and JavaScript.
+
+### **What is pdfkit?**
+
+Pdfkit is an open-source library that generates HTML from web URLs, local files a string and converts this text to PDF format. Web apps often contain tables, charts, and graphs asides text. Export these data professional, PDF converters are the best options, and pdfit converts data regardless of the format, either image, table, graphs, HTML forms, and other complex printable documents.
+
+We will be using wkhtmltopdf alongside pdfkit to achieve perfect work. Wkhtmltopdf is an open-source command-line tool that renders HTML into PDF and various image formats. To know more about wkhtmltopdf see [https://wkhtmltopdf.org/](https://wkhtmltopdf.org/).
+
```
Comment (jumamiller) [inline articles/converting-html-content-to-pdf-with-pdfkit-for-python/index.md:74]: I think the links 're not correctly used, for example, readers would want to click on terms/words they don't understand, to get more information. You don't have to go to the end of the sentence to get the link. My suggestion, have something like this ****We will be using [wkhtmltopdf](https://wkhtmltopdf.org) alongside..........*****.
Make use of this format where applicable throughout your article.

File: articles/converting-html-content-to-pdf-with-pdfkit-for-python/index.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+### **Installing pdfkit**
+
+Pdfkit is installed using the pip command in your terminal.
+
+```bash
+pip install pdfkit
+
+```
+
+![pdfkit-installation](/engineering-education/python-projects-for-beginners/pdfkit.png)
```
Comment (jumamiller) [inline articles/converting-html-content-to-pdf-with-pdfkit-for-python/index.md:86]: What does this image do, tell the user...., for instance, you can say .."as shown in the image below to explain your concept further..."

**Review by jumamiller — CHANGES_REQUESTED** [review]
@Anitaachu, I've suggested a few changes which I realized are repeated severally throughout your article. Kindly work on them.

**Comment by Anitaachu** [comment]
@jumamiller changes have been made, thank you!

**Comment by jumamiller** [comment]
Hi @Anitaachu, I've gone thro' the changes you made. I noticed you only made a single change for each requested change while they are repetitive in different sections of the article. I only pointed a few of the many occurrences in the article. Kindly go thro' the whole article.

**Comment by Anitaachu** [comment]
I have gone through the article and made requested changes @jumamiller

**Comment by jumamiller** [comment]
@hectorkambow, ready for final review on successful plagiarism checks!

**Comment by jumamiller** [comment]
@adrianmurage, you could as well add the 'ready for final review' label...

**Comment by adrianmurage** [comment]
@jumamiller I will 
But the main reason I add them from time to time is that they are needed.
As a reviewer, you should have the permissions to add and remove labels.
If you don't have the necessary permissions you should consult @hectorkambow on Slack preferably.

**Comment by hectorkambow** [comment]
@Anitaachu  👋  good afternoon
Upon doing a final review - I found myself rewording and reworking some of the sentences. 
Some of the paragraphs were a bit difficult to get through - and I believe some sections repeat. 

If you could please revisit this article - be sure to go through, our [resources page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and helpful links on how to improve overall technical writing. 

cc: @jumamiller

**Pull request closed without merging**